src/utils/auth/ping-identity.ts

```diff
@@ -2,6 +2,7 @@ import { OAuthConfig, OAuthUserConfig } from 'next-auth/providers';
 
 // from https://github.com/nextauthjs/next-auth/pull/6614
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 export interface PingProfile extends Record<string, any> {
```

Comment:
`Record<string, string | string[] | number>` ?

Reply:
Changed.
Thank you for the suggestion.